src/lang/base/Syntax.ml

```diff
@@ -257,7 +361,7 @@ module ScillaSyntax (SR : Rep) (ER : Rep) = struct
     | App of ER.rep ident * ER.rep ident list
     | Constr of string * typ list * ER.rep ident list
     | MatchExpr of ER.rep ident * (pattern * expr_annot) list
-    | Builtin of ER.rep ident * ER.rep ident list 
+    | Builtin of builtin * ER.rep ident list
```

Comment:
It seems we are loosing location info here. We could introduce a new type to retain that information:
```
type 'rep builtin_ident =
  | Builtin_ident of builtin * 'rep
[@@deriving sexp]
```